code/game/machinery/doors/airlock.dm

```diff
@@ -202,10 +202,9 @@
 	add_filter("mask_filter", 1, list(type="alpha",icon=mask_file,x=mask_x,y=mask_y))
 
 /obj/machinery/door/airlock/proc/update_other_id()
-	for(var/obj/machinery/door/airlock/A in GLOB.airlocks)
-		if(A.closeOtherId == closeOtherId && A != src)
-			closeOther = A
```

Comment:
From what I can tell this is the only place closeOther is ever given a value, if it's never assigned one get rid of the variable

Reply:
I dont understand. I deleted closeOther from this proc. Its a list now. Removing the list would break this.

Reply:
Oh, I see what you mean.

Unfortunately, closeOther is not deprecated, its used in several checks such as under `/obj/machinery/door/airlock/open`

Reply:
I mean that you removed the only place in the code where `closeOther` is given a value, the other checks you mentioned will always have the same result as `closeOther` will always be null

Reply:
> I mean that you removed the only place in the code where `closeOther` is given a value, the other checks you mentioned will always have the same result as `closeOther` will always be null

Yeah, I see what you mean now.



> Or is that var supposed to get vareditted in the map editor?

Nope, the var isnt redefined at all in any map

I removed it, and after testing, it does seem no longer necessary. Opening/closing doors, cyclelinks, forcing doors, and Airlock Controllers all seem to function without it.

removed in ffa22a7